Manual.md

```diff
@@ -0,0 +1,154 @@
+
+***
+
+# Purpose
+
+The purpose of this Manual is to properly document how `crew` works, create a proper rundown of how and what each function and/or variable does/works. Note that this Manual does not include documentation of functions and variables used in rare cases, such as `case` or `if`, as these are documented in the ruby programming language itself.  
+Note that a understanding of `shell`, `ruby` and the Linux base system should exist before writing packages, as each package script uses such.
```

Comment:
The purpose of this manual is to properly document the `crew` command with a rundown of function and/or variable usage. Note that this manual does not include documentation of ruby functions and variables.  A basic understanding of ruby, shell and Linux is a prerequisite for package development.